src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -114,9 +94,10 @@ public JMenu getImportMenu() {
     private void importBills() {
         System.out.println("imported");
         JFileChooser chooser = new JFileChooser();
-        FileNameExtensionFilter filter = new FileNameExtensionFilter("Your Bills", "csv", "xlsx");
+        //JDialog chooserDialog = chooser.createDialog(mainFrame);
+        FileNameExtensionFilter filter = new FileNameExtensionFilter("Your Bills", "csv");
         chooser.setFileFilter(filter);
-        int ret = chooser.showOpenDialog(this);
+        int ret = chooser.showOpenDialog(mainFrame);
```

Comment:
Good try to make the dialog in the center of the main window!
But there's a better way of doing this.
Later, I can commit to your branch to help you with it.